Typography: Added basic styles for paragraphs and lists

thejdeep opened the pull request.

rohmulan committed:
Typography: Add Link Variables

Fixes gh-119
Closes gh-139

geekman-rohit commented:
@thejdeep Please open a PR on jquery:master from this same branch.
I'll make sure my commits are merged by tomorrow and then we can have everyone review your commits.
Good work!

thejdeep commented:
@geekman-rohit Sure. Will do that right away. Thanks a lot.

thejdeep committed:
Typography: Added styles for paragraphs and lists

Typography: Eliminated need of a mixin for p

Typography: Eliminated need of mixin for p

Typography: Shorthand margins

Typography: Eliminated need of mixin for p

Typography: Fixed indentation

Typography: Fixed default fontSize

Typography: Changed to follow naming conventions

thejdeep committed:
Typography: Added font-size to p map
